f/fluent-bit/fluent-bit_v2.1.10_ubi_8.7.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash -e
+# -----------------------------------------------------------------------------
+#
+# Package	: fluent-bit
+# Version	: v2.1.10
+# Source repo	: https://github.com/fluent/fluent-bit
+# Tested on	: UBI 8.7
+# Language      : C++
+# Travis-Check  : true
+# Script License: Apache License, Version 2 or later
+# Maintainer	: Sumit Dubey <sumit.dubey2@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+trap cleanup ERR
+function cleanup {
+  echo "ERROR: Script failed"
+}
+
+PKG_NAME=fluent-bit
```

Comment:
@sumitd2 Please change variable names in format like PACKAGE_NAME... use PACKAGE in place of PKG. Otherwise I am good with PR for merge